server/src/lib/model/db/language-data/variants.ts

```diff
@@ -505,5 +505,25 @@ export const VARIANTS: Variant[] = [
     locale_name: 'gsw',
     variant_name: 'Hochàlemmànnisch (Hìniga, Pfìrt, usw.)',
     variant_token: 'gsw-FR-hochalem',
-  }
+  },
+  {
+    locale_name: 'ga-IE',
+    variant_name: 'Gaeilge Uladh - Ulster Irish',
```

Comment:
I think we can leave the English translation out, people using the interface should know Irish well enough, and even if not, the word for Irish + U M C should be obvious.